pmm-app/tests/pages/api/securityChecksAPI.js

```diff
@@ -9,8 +9,11 @@ module.exports = {
     for (let i = 0; i < timeout; i++) {
       const results = await this.getSecurityChecksResults();
 
-      if (results.length !== 0) {
-        break;
+      // waiting 10 sec to avoid error "'length' of undefined".
```

Comment:
so where do we wait for 10 seconds?

Reply:
I think we don't need to wait here, just an extra check for `undefined` would be enough

Reply:
I forgot to delete this comment :) I am going to delete it.